container/Dockerfile

```diff
@@ -46,14 +42,23 @@ LABEL org.opencontainers.image.authors="searxng <$GIT_URL>" \
       org.opencontainers.image.version="$SEARXNG_GIT_VERSION"
 
 ENV SEARXNG_VERSION="$SEARXNG_GIT_VERSION" \
-    INSTANCE_NAME="SearXNG" \
-    AUTOCOMPLETE="" \
-    BASE_URL="" \
```

Comment:
I assume that these variables should just be provided as runtime envs, e.g. `SEARXNG_BASE_URL`? I think we should also mention that in the `Environment variables` section of the newly added documentation, i.e. all env variables from https://github.com/searxng/searxng/blob/master/searx/settings_defaults.py#L179-L185.

I don't think we should copy all of them because we'd always need to keep the documentation in sync, but maybe we can reference that file somehow?